src/Element.php

```diff
@@ -248,6 +250,22 @@ public function prop_set_selected(bool $selected):bool {
 		return $this->selected;
 	}
 
+	public function prop_get_form() {
+		if (\in_array($this->tagName, self::$formControlElements)) {
```

Comment:
Not sure what you mean here

Reply:
Using the backslash on functions (`\in_array` vs `in_array) means that other developers would not be able to improve or mock the functionality supplied by that function. Removing the slash means that the function can then be replaced or mocked easier.

Reply:
Aha, OK...I thought that there is no need to override "in_array" function and it would be little bit faster as it does not search for function in current namespace.

Reply:
Maybe there is no need, but I can't think of why we would take action preventing developers from doing so. I think this is an important topic to cover in the styleguide, so I'll type something up tonight. Your feedback and ideas are always welcome.